tests/kuttl/test-suites/default/watcher/01-assert.yaml

```diff
@@ -114,3 +118,16 @@ commands:
       oc exec -n watcher-kuttl-default openstackclient -- openstack service list -f value -c Name -c Type |[ $(grep -c ^watcher) == 1 ]
       SERVICEID=$(oc exec -n watcher-kuttl-default openstackclient -- openstack service list -f value -c Name -c Type -c ID | grep watcher| awk '{print $1}')
       [ $(oc get -n watcher-kuttl-default keystoneservice watcher -o jsonpath={.status.serviceID}) == $SERVICEID ]
+---
+# Check for Container Image environment variables
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      set -euxo pipefail
+      env_variables=$(oc set env $(oc get pods -n openstack-operators -o name -l openstack.org/operator-name=watcher) -n openstack-operators --list)
+      for i in ${env_variables}; do
```

Comment:
would this ever fail? IIUC this will loop over the env vars and grep and echo the ones related to container images, but if those images are not set for some reason, this won't fail, it just won't print anything, maybe we should keep a counter and check at the end that we found 3 `URL_DEFAULT` vars?

Reply:
Done!